s3_to_redshift/ddl/servicebc.history.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE servicebc.history
+(
+  "id" INT NOT NULL ENCODE ZSTD,
+  "user_id" INT ENCODE ZSTD,
+  "title" VARCHAR(255) ENCODE ZSTD,
+  "created_at" TIMESTAMP ENCODE ZSTD,
+  "query_id" INT ENCODE ZSTD,
+  "look_id" INT ENCODE ZSTD,
+  "runtime" DOUBLE PRECISION ENCODE ZSTD,
+  "source" VARCHAR(255) ENCODE ZSTD,
+  "node_id" INT ENCODE ZSTD,
+  "status" VARCHAR(255) ENCODE ZSTD,
+  "slug" VARCHAR(255) ENCODE ZSTD,
+  "cache_key" VARCHAR(255) ENCODE ZSTD,
+  "result_source" VARCHAR(255) ENCODE ZSTD,
+  "message" VARCHAR(255) ENCODE ZSTD,
+  "connection_name" VARCHAR(255) ENCODE ZSTD,
+  "connection_id" VARCHAR(255) ENCODE ZSTD,
+  "dialect" VARCHAR(255) ENCODE ZSTD,
+  "completed_at" TIMESTAMP ENCODE ZSTD,
+  "force_production" INT ENCODE ZSTD,
+  "generate_links" INT ENCODE ZSTD,
+  "path_prefix_id" INT ENCODE ZSTD,
+  "cache" INT ENCODE ZSTD,
+  "cache_only" INT ENCODE ZSTD,
+  "sql_query_id" INT ENCODE ZSTD,
+  "render_key" VARCHAR(255) ENCODE ZSTD,
+  "rebuild_pdts" INT ENCODE ZSTD,
+  "server_table_calcs" INT ENCODE ZSTD,
+  "dashboard_id" VARCHAR(255) ENCODE ZSTD,
+  "result_format" VARCHAR(255) ENCODE ZSTD,
+  "apply_formatting" INT ENCODE ZSTD,
+  "dashboard_session" VARCHAR(255) ENCODE ZSTD,
+  "apply_vis" INT ENCODE ZSTD,
+  "models_dir" VARCHAR(255) ENCODE ZSTD,
+  "workspace_id" VARCHAR(255) ENCODE ZSTD,
+  "result_maker_id" INT ENCODE ZSTD
+);
+
```

Comment:
@BJenning We might want to set the sort key for the history table to "created_at". This table may see enough growth over time to warrant it.